index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="styles.css">
```

Comment:
- [ ] kindly consider linking all your HTML files into one cause currently your pop-ups are being opened in a new tab which is not giving a good user experience
<img width="1136" alt="Screen Shot 2023-06-28 at 09 29 25" src="https://github.com/binyamolango/Personal-Portfolio/assets/60146030/4607e40d-3491-420a-8bec-2ce63f9cf614">
 kindly fix that to give a good user experience